packages/puppeteer-core/src/api/Input.ts

```diff
@@ -469,7 +469,22 @@ export abstract class Mouse {
     options?: {delay?: number},
   ): Promise<void>;
 }
-
+/**
+ * The Touch interface exposes methods to manipulate touches that have been started
+ * @public
+ */
+export interface Touch {
```

Comment:
I think we may need to change the name as it clashes with the DOM Touch interface, that is why the `.md` file reference `Touch_2` . I will try to think of something

Reply:
TouchPoint? TouchPointer? Pointer?

Reply:
How about TouchHandle?

Reply:
I like it.

Reply:
I'll change it to TouchHandle then, hang on